src/utils.js

```diff
@@ -1,4 +1,4 @@
-import { registerApplication } from 'single-spa';
+// import { registerApplication } from 'single-spa';
```

Comment:
I'm not sure why the code below is commented out, but I commented out this import too. If you need to bring back this import, we'll have to figure out how to do it without creating duplicate instances of singleSpa.

Reply:
I will remove it after merge